mealplanner-ui/src/pages/MealPlans/MealPlans.tsx

```diff
@@ -140,6 +142,18 @@ const MealPlanCard = (props: MealPlanCardProps) => {
               >
                 <DeleteTwoTone />
               </IconButton>
+              {getCurrentPerson().personName === "Admin" || getCurrentPerson().personName === "Meal Designer" ? (
```

Comment:
We should be checking for the role and not the name, as names can differ depending on who is assigned as admins and meal designers. You need to check whether the role is "app_admin" and "app_meal_designer" here. You need to include the role in the graphql query 
```
query currentPerson {
  currentPerson {
    fullName
    email
    role
  }
}

Then, set the relay record in the setCurrentUser function.